projects/packages/waf/package.json

```
@@ -0,0 +1,31 @@
+{
+	"private": true,
+	"name": "@automattic/jetpack-waf",
+	"version": "0.1.0-alpha",
+	"description": "Tools to assist with the Jetpack Web Application Firewall",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "echo 'Not implemented.",
+		"build-js": "echo 'Not implemented.",
+		"build-production": "echo 'Not implemented.",
+		"build-production-js": "echo 'Not implemented.",
+		"clean": "true",
+		"distclean": "rm -rf node_modules && pnpm run clean",
+		"install-if-deps-outdated": "pnpm install --no-prod --frozen-lockfile"
```

Remove this to match #22390
```suggestion
		"clean": "true"
```
Or just remove the whole file if there isn't going to be any JS in the package.